mbofui/app/index.html

```diff
@@ -1,95 +1,150 @@
 <!DOCTYPE html>
-<html ng-app="mbofuiApp">
+<html ng-app="ocellus">
```

Comment:
If the internal name of the app changes, shouldn't the directory names `mbofui` and `mbof` and references to them be changed, too?